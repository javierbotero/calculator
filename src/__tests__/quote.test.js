import React from 'react';
import { screen, render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Quote from '../components/quote';

describe('Quote Tests', () => {
  it('Makes sure the text "Mathematics is not about numbers" is added', () => {
    render(<Quote />);
    expect(screen.getByText('Mathematics is not about numbers', { exact: false }))
      .toBeTruthy();
  });

  it('Checks match with the snapshot', () => {
    const tree = renderer
      .create(<Quote />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
