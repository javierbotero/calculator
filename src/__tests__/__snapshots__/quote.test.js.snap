// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Tests Checks match with the snapshot 1`] = `
<h3
  className="quote"
>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
</h3>
`;
